test/testInteraction.cpp

```diff
@@ -718,6 +724,8 @@ TEST(EMPairProduction, allBackgrounds) {
 	em.setPhotonField(ebl);
 	ebl = new IRB_Stecker16_lower();
 	em.setPhotonField(ebl);
+	ebl = new IRB_Finke22();
```

Comment:
Why is it called EBL here and IRB else?

Reply:
I think that depends on who implemented the test in the first place. Could be cleaned up in the future.